+tests/TestSetupMYM.m

```diff
@@ -0,0 +1,117 @@
+
+classdef TestSetupMYM < tests.Prep
+% TestSetupMyM tests setupMYM
+
+    methods (Test)
+        function TestSetupMYM_testDefaultInstall(testCase)
+
+            [dir, nam, ext] = fileparts(mfilename('fullpath'));
+            tmym = strcat(dir, '/../', 'mym');
+
+            if isdir(tmym)
+                fprintf('testDefaultInstall: mymdir %s exists. removing\n', ...
+                        tmym);
+                rmdir(tmym, 's');
+            end
+            
+            ms = setupMYM();
+            testCase.verifyTrue(strcmp(ms, 'master'));
+            testCase.verifyTrue(isdir(tmym));
+
+        end
+        function TestSetupMYM_testVersionInstallFresh(testCase)
+            [dir, nam, ext] = fileparts(mfilename('fullpath'));
+            tmym = strcat(dir, '/../', 'mym');
+
+            if isdir(tmym)
+                fprintf('testVersionInstallFresh: removing mymdir %s\n', ...
+                        tmym);
+                rmdir(tmym, 's');
+            end
+
+            % TODO: how manage version string?
+            ms = setupMYM('2.7.2');
+            testCase.verifyTrue(strcmp(ms, '2.7.2'));
+            testCase.verifyTrue(isdir(tmym));
+
+        end
+        function TestSetupMYM_testVersionInstallStale(testCase)
```

Comment:
Not too clear on the objective of the below tests. Thinking you might only need 2 tests to cover most cases:

1. Install master, verify, install specific version w/o force, verify error, install specific version w/ force, verify
2. Install specific version, verify, install master w/o force, verify error, install master w/ force, verify